addon/helpers/without.js

```diff
@@ -16,7 +15,7 @@ export function without(needle, haystack) {
     return false;
   }
 
-  if (typeOf(needle) === 'array' && get(needle, 'length')) {
+  if (isEmberArray(needle) && get(needle, 'length')) {
```

Comment:
Will this break for normal arrays? I.e. an array in a project where prototypes are not extended by default?